types/spectest/tests/consensusdata/capella_blinded_block_validation.go

```diff
@@ -1,6 +1,16 @@
 package consensusdata
 
+import (
+	"github.com/attestantio/go-eth2-client/spec"
+	"github.com/bloxapp/ssv-spec/types/testingutils"
+)
+
 // CapellaBlindedBlockValidation tests a valid consensus data with capella blinded block
-func CapellaBlindedBlockValidation() *SpecTest {
-	panic("implement")
+func CapellaBlindedBlockValidation() *ConsensusDataTest {
```

Comment:
In Deneb PR we should maybe add tests if they are not there?

Reply:
Yes, we will